app/views/declaration/exporter_details.scala.html

```diff
@@ -21,49 +21,150 @@
 @import views.Title
 @import models.requests.JourneyRequest
 @import controllers.navigation.Navigator
+@import services.Countries
+@import scala.collection.immutable
 
-@this(main_template: views.html.main_template)
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.html.components.gds._
+@import services.view.AutoCompleteItem
+@import views.BackButton
+@import play.twirl.api.HtmlFormat
+@import views.components.gds.Styles
+
+@this(
+    govukLayout: gdsMainTemplate,
+    govukButton: GovukButton,
+    govukRadios: GovukRadios,
+    govukDetails : GovukDetails,
+    govukFieldset: GovukFieldset,
+    errorSummary: errorSummary,
+    sectionHeader: sectionHeader,
+    saveButtons: saveButtons,
+    tariffDetails: tariffDetails,
+    exportsInputText: exportsInputText,
+    pageTitle: pageTitle,
+    formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @(mode: Mode, form: Form[ExporterDetails])(implicit request: JourneyRequest[_], messages: Messages)
 
 @tariffBodyDesc = { @Html(messages("declaration.type.consignmentTariffText", components.details_content_link())) }
 
-@tariffBody = { @components.tariff_body_partial(messages("supplementary.exporter-details.help-item1"), messages("supplementary.exporter-details.help-item2"))(tariffBodyDesc)}
+@tariffBody = {
+    @components.tariff_body_partial(
+                    messages("supplementary.exporter-details.help-item1"),
+                    messages("supplementary.exporter-details.help-item2")
+                )(tariffBodyDesc)
+}
+
+@tariff = { @components.details(messages("site.details.summary_text_these")){ @tariffBody } }
+
+@pageHint = {
+    <span class="govuk-hint">
+        @messages("supplementary.consignor.title.hint")
+    </span>
+}
+
+@exportsName = {
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+        @messages("supplementary.exporter-details-name-address")
+    </legend>
+}
+
+@or = {
+    <p class="govuk-body">Or</p>
+}
+
+@autocomplete = {
+    @components.fields.field_accessible_autocomplete(
+        form("details.address.country"),
+        label = messages("supplementary.address.country"),
+        None,
+        None,
+        messages("declaration.originationCountry.empty"),
+        AutoCompleteItem.fromCountry(Countries.allCountries, _.countryCode)
+    )
+}
+
+@fullName = {
```

Comment:
I thought we were supposed to use @iservport address details approach. 
Please check it out.

Reply:
@iservport's approach has been utilised